pkgs/by-name/dr/druid/package.nix

```
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, extensions ? { }
+, libJars ? [ ]
+, nixosTests
+, mysqlSupport ? true
+}:
+let
+  inherit (lib) concatStringsSep mapAttrsToList optionalString forEach;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache-druid";
+  version = "29.0.0";
+  src = fetchurl {
+    url = "mirror://apache/druid/${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}-bin.tar.gz";
+    sha256 = "sha256-Ka/coMWYXDjP18hgwwJq+xzSO7tL7/XtIZRcekpX6g4=";
+  };
+  mysqlConnector = fetchurl {
+    url = "https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.48/mysql-connector-java-5.1.48.jar";
+    hash = "sha256:1a7cnr4r03mv7j0d6sji07m892ygcr7wgya4mzj5l7w2lfm6rqjn";
```

please use SRI hash:

```
$ nix hash to-sri --type sha256 'sha256:1a7cnr4r03mv7j0d6sji07m892ygcr7wgya4mzj5l7w2lfm6rqjn'
sha256-VuJsqqOCH1rkr0T5x09mz4uE6gFRatOAPLsOkEm27Kg=
```

```suggestion
    hash = "sha256-VuJsqqOCH1rkr0T5x09mz4uE6gFRatOAPLsOkEm27Kg=";
```